server/main.js

```diff
@@ -9,33 +9,36 @@ Meteor.startup(() => {
   // Temporary template player for first run
   // You can use this to edit in the database
   if (Players.find().count() === 0) {
-		Players.insert(
-  		{
-  			name: "DELETE_ME",
-  			hongKongLeagueName: "HK_DELETE_ME",
-  			hongKongElo: 0, //added
-  			hongKongGamesPlayed: 0, //added
-  			hongKongPositionSum: 0, //added
-  			hongKongHandsWin: 0, //added
-  			hongKongHandsLose: 0, //added
-  			hongKongHandsTotal: 0, //added
-  			hongKongWinPointsTotal: 0, //added
-  			hongKongChomboTotal: 0, //added
-  			hongKongBankruptTotal: 0, //added
+	  for (let i = 0; i < 4; i++) {
```

Comment:
Remove this please.  It's easy enough to add local test players manually through mongo CLI.